AdminGuide.md

```diff
@@ -0,0 +1,156 @@
+# Admin Guide for WikiEduDashboardTools
```

Comment:
WikiEduDashboardTools is one particular service, not the main Dashboard server, so that name probably shouldn't be in the main header here.